_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -296,6 +296,52 @@ Use the following steps to register a snapshot repository backed by an Azure sto
    }
    ```
 
+### Microsoft azure
+
+1. To use a Microsoft Azure Blob Storage as a snapshot repository, install the `repository-azure` plugin on all nodes:
+
+   ```bash
+   ./bin/opensearch-plugin install repository-azure
+   ```
+
+1. To use `repository-azure` plugin after installation, you need to define your azure storage settings before
+   initialising the node:
+
+   1. Define your azure storage account name in secure setting:
```

Comment:
Capitalization of Azure and Azure Managed Identities?

Reply:
fixed by @Naarcha-AWS